lib/ui/component/alert_filter_bar.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+
+class AlertFilterBar extends StatefulWidget {
+  final Function(String val) onFilterChanged;
+  final Function(String val) onOrderChanged;
+
+  AlertFilterBar({super.key, required this.onFilterChanged, required this.onOrderChanged});
+
+  @override
+  _AlertFilterBarState createState() => _AlertFilterBarState();
```

Comment:
ide gives me warning here not to use private func for public api

Reply:
Done, return type should be `State<AlertFilterBar>`